winterfell/webapp/src/js/formRenderingService.js

```diff
@@ -27,3 +27,29 @@ app.factory('formRenderingService', ['$window', '$http', function ($window, $htt
         }
     }
 }]);
+
+/**
+ * Service to retrieve the formly template (which is just a static JSON file)
+ * for a given form.
+ */
+app.factory('formTemplateService', ['$http', function ($http) {
```

Comment:
Which library is that, I don't see anything called net in the package json? I agree $http is a little kludgy, I think using $resource might be preferred.